python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -387,6 +387,13 @@ class SensorResult(
             AddDynamicPartitionsRequest]]]): A list of dynamic partition requests to request dynamic
             partition addition and deletion. Run requests will be evaluated using the state of the
             partitions with these changes applied.
+        asset_events (Optional[Sequence[Union[AssetObservation, AssetMaterialization, AssetCheckEvaluation]]]): A
+            list of materializations, observations, and asset check evaluations that the system
+            will persist on your behalf at the end of sensor evaluation. These events will be not
+            be associated with any particular run, but will be queryable and viewable in the asset catalog.
+            Experimental.
```

Comment:
```suggestion
        asset_events (Optional[Sequence[Union[AssetObservation, AssetMaterialization, AssetCheckEvaluation]]]):  (Experimental) A
            list of materializations, observations, and asset check evaluations that the system
            will persist on your behalf at the end of sensor evaluation. These events will be not
            be associated with any particular run, but will be queryable and viewable in the asset catalog.
```

we usually start with the "Experimental" note, such as [here](https://github.com/dagster-io/dagster/blob/ce270a8be47c34c8968a361200566172d2a1383d/python_modules/dagster/dagster/_core/types/config_schema.py#L153-L159)